try.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
yes, it is a convenient script that I always use to build and try, and I got tired of recreating it every time I use a fresh image.